src/Eye4web/ZfcUser/Settings/Factory/Mapper/DoctrineORM/UserSettingMapperFactory.php

```diff
@@ -32,7 +32,7 @@ class UserSettingMapperFactory implements FactoryInterface
      */
     public function createService(ServiceLocatorInterface $serviceLocator)
     {
-        $entityManager = $serviceLocator->get('zfcuser_doctrine_em');
+        $entityManager = $serviceLocator->get('Doctrine\ORM\EntityManager');
```

Comment:
Changed passed Service Name to <code>Doctrine\ORM\EntityManager</code> as there is no service named <code>zfcuser_doctrine_em</code>